inst/htmljs/animint.js

```diff
@@ -125,24 +125,61 @@ var animint = function (to_select, json_file) {
   var measureText = function(pText, pFontSize, pAngle, pStyle) {
     if (pText === undefined || pText === null || pText.length === 0) return {height: 0, width: 0};
     if (pAngle === null || isNaN(pAngle)) pAngle = 0;
-
+    
+    // Create temporary container to measure text
     var container = element.append('svg');
-    // do we need to set the class so that styling is applied?
-    //.attr('class', classname);
-
-    container.append('text')
-      .attr({x: -1000, y: -1000})
+    var textElement = container.append('text')
       .attr("transform", "rotate(" + pAngle + ")")
       .attr("style", pStyle)
-      .attr("font-size", pFontSize)
-      .text(pText);
-
+      .attr("font-size", pFontSize);
+    
+    // Check if text contains <br/> tags (multi-line)
+    var textStr = String(pText || '');
+    var lines = textStr.split('<br/>');
+    
+    // Always use setMultilineText for consistent rendering
+    setMultilineText(textElement, pText);
+    
+    // Get bounding box after rendering
     var bbox = container.node().getBBox();
+    
+    // Clean up temporary element
     container.remove();
-
+    
+    // Return measured dimensions
     return {height: bbox.height, width: bbox.width};
   };
 
+
+  /**
+   * Set multi-line text on SVG text elements
+   * Converts <br/> tags to <tspan> elements for proper SVG rendering
+   */
+  var setMultilineText = function(textElement, text) {
+                textElement.each(function(d) {
```

Comment:
is this each necessary? can it be removed?

is the setMultilineText necessary as a separate function from measureText? can they be combined?